assets/js/components/Loadpoint.vue

```diff
@@ -95,34 +100,62 @@
 
 		<div class="row">
 			<div class="col-12 col-md-4 mt-3 mb-3 mb-md-0">
-				<Vehicle v-bind:state="state"></Vehicle>
+				<Vehicle
+					:charging="charging"
+					:connected="connected"
+					:minSoC="minSoC"
+					:soc="soc"
+					:socCharge="socCharge"
+					:socTitle="socTitle"
+				></Vehicle>
 			</div>
 
 			<div class="col-12 col-md-4 d-none d-md-block mt-3" v-if="multi">
 				<div class="mb-2">Modus</div>
 				<Mode
 					class="btn-group-sm"
-					v-bind:mode="state.mode"
-					:pv="pv"
-					v-on:updated="targetMode"
+					:mode="mode"
+					:pvConfigured="pvConfigured"
+					v-on:updated="setTargetMode"
 				></Mode>
 			</div>
 			<div class="col-12 col-md-4 d-none d-md-block mt-3" v-if="multi && hasTargetSoC">
 				<div class="mb-2">Ladeziel</div>
 				<Soc
 					class="btn-group-sm"
-					v-bind:soc="state.targetSoC"
-					:levels="state.socLevels"
-					v-on:updated="targetSoC"
+					:soc="targetSoC"
+					:levels="socLevels"
+					v-on:updated="setTargetSoC"
 				></Soc>
 			</div>
 
 			<div class="col-md-8 d-none d-md-block" v-if="!multi">
-				<LoadpointDetails v-bind:state="state"></LoadpointDetails>
+				<LoadpointDetails
+					:connected="connected"
+					:minSoC="minSoC"
+					:socCharge="socCharge"
+					:climater="climater"
+					:chargePower="chargePower"
+					:chargedEnergy="chargedEnergy"
+					:range="range"
+					:chargeDuration="chargeDurationDisplayed"
+					:soc="soc"
+					:chargeEstimate="chargeEstimate"
+				></LoadpointDetails>
 			</div>
 
 			<div class="col-12 d-md-none">
-				<LoadpointDetails v-bind:state="state"></LoadpointDetails>
+				<LoadpointDetails
+					:connected="connected"
+					:minSoC="minSoC"
+					:socCharge="socCharge"
+					:climater="climater"
+					:chargePower="chargePower"
+					:chargedEnergy="chargedEnergy"
+					:range="range"
+					:chargeDuration="chargeDurationDisplayed"
+					:chargeEstimate="chargeEstimate"
```

Comment:
Würde vorschlagen diese Werte alle in einem Objekt zusammenzufassen. Die Loadpoint Komponente kann das Objekt dann direkt an die LoadpointDetails Komponente binden, muss aber selbst gar nicht genau wissen wie das Objekt aufgebaut ist.

```
<LoadpointDetails v-bind="details" />
```